FirebaseMessaging/Sources/Public/FIRMessaging.h

```diff
@@ -504,6 +536,20 @@ NS_SWIFT_NAME(Messaging)
  */
 - (FIRMessagingMessageInfo *)appDidReceiveMessage:(NSDictionary *)message;
 
+#pragma mark - GDPR
+/**
+ * Deletes all the tokens and checkin data of the Firebase project  and related data on the server
+ * side. A network connection is required for the method to succeed.
+ *
+ * This does not delete the Firebase Installations ID. See `FIRInstallations deleteWithCompletion:` for deleting that.
```

Comment:
If I remember correctly, we should prefer Swift variants of API in the doc string because we have more devs using Firebase from Swift and this amount is growing. So it should be `Installations.delete(completion:)` instead of `FIRInstallations deleteWithCompletion:`.

@ryanwilson @paulb777 Could you confirm please.

Reply:
Confirmed. If only one is listed it should be the Swift API.